bin/brew

```diff
@@ -44,4 +44,17 @@ fi
 
 HOMEBREW_LIBRARY="$HOMEBREW_REPOSITORY/Library"
 
-source "$HOMEBREW_LIBRARY/Homebrew/brew.sh"
+if [[ -n "$HOMEBREW_ENV_FILTERING" ]]
+then
+  PATH="/usr/bin:/bin:/usr/sbin:/sbin"
```

Comment:
@zmwangx Still necessary for i.e. not breaking `brew update` by putting stupid things like `head` in your PATH. This will break a bunch of this, I realise, but it's the first step to getting this actually working.